<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CAMBioMed Preprocessor</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="menuFile_NewProject"/>
    <addaction name="menuFile_OpenProject"/>
    <addaction name="menuFile_SaveProject"/>
    <addaction name="menuFile_SaveProjectAs"/>
    <addaction name="menuFile_CloseProject"/>
    <addaction name="separator"/>
    <addaction name="menuFile_ExportAll"/>
    <addaction name="separator"/>
    <addaction name="menuFile_Exit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="menuHelp_About"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="menuView_ShowOriginal"/>
    <addaction name="menuView_ShowLensCorrected"/>
    <addaction name="menuView_ShowColorCorrected"/>
    <addaction name="menuView_ShowPerspectiveCorrected"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="menuEdit_DetectQuadrat"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="menuFile_OpenProject"/>
   <addaction name="separator"/>
   <addaction name="menuEdit_DetectQuadrat"/>
   <addaction name="menuView_ShowOriginal"/>
   <addaction name="menuView_ShowLensCorrected"/>
   <addaction name="menuView_ShowColorCorrected"/>
   <addaction name="menuView_ShowPerspectiveCorrected"/>
  </widget>
  <action name="menuFile_OpenProject">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>&amp;Open Project...</string>
   </property>
   <property name="statusTip">
    <string>Open an existing project.</string>
   </property>
  </action>
  <action name="menuFile_Exit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="statusTip">
    <string>Exit the application.</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::QuitRole</enum>
   </property>
  </action>
  <action name="menuHelp_About">
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="statusTip">
    <string>About this application.</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::AboutRole</enum>
   </property>
  </action>
  <action name="menuView_ShowOriginal">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::InsertImage"/>
   </property>
   <property name="text">
    <string>Show &amp;Original</string>
   </property>
  </action>
  <action name="menuView_ShowLensCorrected">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::InsertImage"/>
   </property>
   <property name="text">
    <string>Show &amp;Lens Corrected</string>
   </property>
  </action>
  <action name="menuView_ShowColorCorrected">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::InsertImage"/>
   </property>
   <property name="text">
    <string>Show &amp;Color Corrected</string>
   </property>
  </action>
  <action name="menuView_ShowPerspectiveCorrected">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::InsertImage"/>
   </property>
   <property name="text">
    <string>Show &amp;Perspective Corrected</string>
   </property>
  </action>
  <action name="menuFile_NewProject">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
   </property>
   <property name="text">
    <string>&amp;New Project...</string>
   </property>
   <property name="statusTip">
    <string>Create a new project.</string>
   </property>
  </action>
  <action name="menuFile_SaveProject">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
   </property>
   <property name="text">
    <string>&amp;Save Project</string>
   </property>
   <property name="statusTip">
    <string>Save the current project.</string>
   </property>
  </action>
  <action name="menuFile_SaveProjectAs">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSaveAs"/>
   </property>
   <property name="text">
    <string>Save Project &amp;As...</string>
   </property>
   <property name="statusTip">
    <string>Save the current project under a new name.</string>
   </property>
  </action>
  <action name="menuFile_CloseProject">
   <property name="text">
    <string>&amp;Close Project</string>
   </property>
   <property name="statusTip">
    <string>Close the current project.</string>
   </property>
  </action>
  <action name="menuFile_ExportAll">
   <property name="text">
    <string>E&amp;xport All...</string>
   </property>
  </action>
  <action name="menuEdit_DetectQuadrat">
   <property name="text">
    <string>Detect &amp;quadrat</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
